src/app/services/register.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { environment } from 'src/environments/environment';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RegisterService {
+
+  constructor(private http: HttpClient) { }
+
+  userRegister(user) {
```

Comment:
Same as above, create an interface (AKA model) and tell the function which kind of data it's going to return